qa/run_xfstests.sh

```diff
@@ -46,7 +51,7 @@ TESTS="-g auto"	# The "auto" group is supposed to be "known good"
 
 # rbd presents geometry information that causes mkfs.xfs to
 # issue a warning.  This option avoids this class of problems.
-XFS_MKFS_OPTIONS="-l su=32k"
+#XFS_MKFS_OPTIONS="-l su=32k"
```

Comment:
it makes me think we might want to add these just for the subset of tests that depend on them in a future update